### ISSUE TITLE
Open hardware challenge 2023

### ISSUE THREAD

**Issue by SanliFaez** [delftopenhardware/openhardwarenl#5]
To motivate and raise awareness about the value of open hardware, we can organize a national challenge in which student teams receive instruction and mentorship to tackle a challenging project, such as building a battery or contributing to the energy transition.

**Comment by SanliFaez** [comment]
@Delft-Open-Hardware can you please open another project to couple with all the issues regarding the OH challenge?

**Comment by Vsaggiomo** [comment]
Batteries may be too niche, though, and I'm not sure how many universities will participate

**Comment by SanliFaez** [comment]
I think Batteries, specially electrochemical ones are multi-disciplinary and a very good candidate for sciency project. TUE, TUD, RUG, UU, and UT will definitely participate, and perhaps some technical hoogscholen. Another similar project can be passive cooling. In general, I think connection with sustainability and energy will make such a project more attractive

**Comment by Vsaggiomo** [comment]
Yes, the TU ones and UU/RUG, where battery projects are already running.
Doubt you'll get participants from WUR, UVA, VU, Rotterdam, Leiden, Maastricht, and Nijmegen... which incidentally are the ones you want to bring on board for OH. 

I think there are two options: 
A free for all, like iGem, where the project have few restrictions on what and how to do it. 
Or a focused challenge, like [SensUs](https://www.sensus.org), where you have a ton of restriction for developing a sensor.

**Comment by Jerzeek** [comment]
> @Delft-Open-Hardware can you please open another project to couple with all the issues regarding the OH challenge?

@santoshilam Will pick this up :)

**Comment by SanliFaez** [comment]
here's a first sketch for the FAIR battery challenge 
https://github.com/delftopenhardware/openhardwarenl/blob/position_paper/FAIRbattery_challenge.md

I posted it to trigger the discussion on assessing the open hardware challenges from the perspective of collaboration and usefulness, instead of merely performance. suggestions on evaluation and rewarding are especially welcome!
